client/modules/User/components/DashboardTabSwitcher.jsx

```diff
@@ -31,8 +38,23 @@ Tab.propTypes = {
   to: PropTypes.string.isRequired
 };
 
+// It is good for right now, because we need to separate the nav dropdown logic from the navBar before we can use it here
```

Comment:
Can we make a todo list for future refactorings?  Maybe add a section to that Google doc?

Reply:
yeah we should, and we can do it in github instead of google doc, we shoul create a todo list in github